### PR TITLE
Update Deep_Learning_Project-Pytorch.ipynb

### PR DESCRIPTION
**Pull request by bobbleoxs**
address two small error messages: 

1. invalid index of a 0-dim tensor;
2. implicit dimension choice for softmax

### CODE REVIEW

**Comment by bobbleoxs** [comment]
ditto
